### ISSUE TITLE
Vimeo / Soundcloud

### ISSUE THREAD

**Issue by andreasarnol02** [41studio/discjoke#8]
This would be perfect if we can add songs from vimeo or soundcloud.

**Comment by aditiamahdar** [comment]
yes, actually we've planned to make this app available to add song from dailymotion and vimeo but it's not done yet and we've our hand busy currently :lol:

**Comment by andreasarnol02** [comment]
Perfect!
Let me know when this feature published ;)
